legend-depot-artifacts-refresh/src/main/java/org/finos/legend/depot/store/artifacts/services/file/FileGenerationHandlerImpl.java

```diff
@@ -117,7 +118,8 @@ public MetadataEventResponse refreshProjectVersionArtifacts(StoreProjectData pro
                     {
                         String elementPath = entityMap.get(entityPath.get()).getPath();
                         FileGeneration generation = new FileGeneration(generatedFile.getPath(), generatedFile.getContent());
-                        generations.createOrUpdate(new StoredFileGeneration(projectData.getGroupId(), projectData.getArtifactId(), versionId, elementPath, null, generation));
+                        String generationType = this.getExtensionKeyFromGeneration(generation.getPath(), entityPath.get());
```

Comment:
so here check if the type is present... if not log an error in the response and set the type to something like UNKNOWN or similar so we can query those too